cobra/test/data/update_pickles.py

```diff
@@ -60,8 +60,8 @@
 mini.add_reaction(ecoli_model.reactions.EX_lac__D_e.copy())
 r = cobra.Reaction("D_LACt2")
 mini.add_reaction(r)
-r.gene_reaction_rule = ecoli_model.reactions.D__LACt2pp.gene_reaction_rule
-r.reaction = ecoli_model.reactions.D__LACt2pp.reaction.replace("_p", "_e")
+# r.gene_reaction_rule = ecoli_model.reactions.D__LACt2pp.gene_reaction_rule
+# r.reaction = ecoli_model.reactions.D__LACt2pp.reaction.replace("_p", "_e")
```

Comment:
If these replacements are no longer required to satisfy the old existing tests, please just delete the lines entirely.